src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -375,6 +413,24 @@ def check_inputs(self, prompt, height, width, callback_steps):
                 f" {type(callback_steps)}."
             )
 
+        if prompt is not None and prompt_embeds is not None:
+            raise ValueError(
+                f"Cannot forward both `prompt`: {prompt} and `prompt_embeds`: {prompt_embeds}. Please make sure to"
+                " only forward one of the two."
```

Comment:
maybe 
```suggestion
                "You can only specify one of `prompt` and `prompt_embeds`. Please make sure to only specify one of them."
```

Think it's better to have shorter errors.

Reply:
Disagree here, people will recognize easily what they have passed, so think it's always good to be specific